src/app/debug/table/table.component.html

```diff
@@ -1,113 +1,133 @@
-    <app-button text="Open In New Tab" *ngIf="showOpenInTabButton()" (click)="openReportInTab()"></app-button>
-    <app-active-filters class="ml-2" [activeFilters]="currentFilters" (click)="toggleFilter()"></app-active-filters>
-  </div>
-  <div [hidden]="doneRetrieving">
-    <mat-spinner mode="indeterminate" [diameter]="30"></mat-spinner>
-  </div>
-  <div data-cy-debug="table" [hidden]="!doneRetrieving" class="table-responsive" id="metadataTable">
-    <table class="table mb-0" matSort (matSortChange)="helperService.sortData($event, tableSettings.reportMetadata)" [style.font-size]="getFontSize()">
-      <thead id="table-header">
-        <tr>
-          <th class="table-row-checkbox header-padding">
-            <input data-cy-debug="selectAll" type="checkbox" class="vertical-middle" [checked]="allRowsSelected" (click)="selectAllRows()" title="Select all rows" [style.width]="getCheckBoxSize()" [style.height]="getCheckBoxSize()">
-          </th>
-          <th class="table-id-cell header-cell header-padding"
-              *ngFor="let header of viewSettings.currentView.metadataLabels; let i = index"
-              [attr.mat-sort-header]="i"
-              [title]="header">
-            {{ getShortenedTableHeaderNames(header) }}
-          </th>
-        </tr>
-      </thead>
-      <tbody data-cy-debug="tableBody">
-        <tr class="table-row" *ngFor="let row of tableSettings.reportMetadata.slice(0, tableSettings.displayAmount); let i = index"
-            (click)="highLightRow(i);"
-            [ngClass]="{'highlight': selectedRow === i}"
-            [style.background-color]="getStatusColor(row)"
-            [attr.data-cy-record-table-index]=i
-        >
-          <td class="table-row-checkbox" [style.padding]="getTableSpacing()">
-            <input type="checkbox" [checked]="row.checked" (click)="toggleCheck(row)" [style.width]="getCheckBoxSize()" [style.height]="getCheckBoxSize()">
-          </td>
-          <td
-            *ngFor="let metadataName of viewSettings.currentView.metadataNames"
-            [style.padding]="getTableSpacing()"
-            class="table-cell"
-            (click)="openReport(row.storageId)"
-            [title]="row[metadataName]">
-            {{ row[metadataName]|tableCellShortener }}</td>
-        </tr>
-      </tbody>
-    </table>
+      @switch (getAmountOfReportsSelected()) {
+        @case (1) {
+          <app-button text="Open In New Tab" (click)="openReportInTab()"></app-button>
+        }
+        @case (2) {
+          <app-button text="Compare" title="Compare" (click)="compareTwoReports()"></app-button>
+        }
+      }
+      <app-active-filters class="ml-2" [activeFilters]="currentFilters" (click)="toggleFilter()"></app-active-filters>
+    </div>
+    @if (!doneRetrieving) {
+      <div>
+        <mat-spinner mode="indeterminate" [diameter]="30"></mat-spinner>
+      </div>
+    }
+    @if (doneRetrieving) {
```

Comment:
Change this to an @if @else:
```suggestion
    @if (!doneRetrieving) {
      <div>
        <mat-spinner mode="indeterminate" [diameter]="30"></mat-spinner>
      </div>
    }
    @else {
```
I'd actually prefer having the spinner at the bottom in the @else

Reply:
Ah that slipt my mind. I added the change.